odr-vm-pvc-metro/source.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: VolumeImportSource
+metadata:
+  name: cirros-source
+spec:
+  source:
+    registry:
+      # Image customized for ramen testing.
+      url: "docker://quay.io/nirsof/cirros:0.6.2-1"
```

Comment:
Do you have a Dockerfile for this image that can be included so others can recreate this custom cirros image?

Reply:
It is part of ramen code here:
https://github.com/RamenDR/ramen/tree/main/test/vms/cirros

The logger service it uses is here:
https://github.com/RamenDR/ramen/tree/main/test/vms/ramen

We can migrate it out of ramen, and add it to this repo.